TGC/common/buildings.txt

```diff
@@ -750,9 +750,10 @@
 		province = yes
 		# Audax Validator "." Ignore_NEXT
 		one_per_state = yes
+		pop_build_factory = yes
```

Comment:
Missing line perhaps?

Reply:
No, intentional. I've balanced around that